app/src/main/java/com/github/dedis/student20_pop/model/network/level/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Classes used to model the levels of the protocol
+ **/
+package com.github.dedis.student20_pop.model.network.level;
```

Comment:
Could you also do a package-info for level.low, level.mid and level.high?

Reply:
Done